osgpkitools/ExceptionDefinitions.py

```diff
@@ -0,0 +1,207 @@
+""" This script contains all the exception classes and would be exclusively used for handling exceptions"""
```

Comment:
I was really hoping to never see this file again. Have you removed any exceptions that you're not actively using?

Reply:
I'm not a fan of making an exception class for every type of error either, but I guess Jeny was just following examples.  She had not removed unused ones, but I will do that.